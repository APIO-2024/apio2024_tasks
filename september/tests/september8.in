txy4h26c1rm1uv8tr3eonahd67u8h56x
4
7 1
0 0 1 2 4 5
6 3 5 1 2 4
10 1
0 0 2 2 3 4 5 6 8
9 7 8 6 5 4 1 3 2
7 1
0 1 1 2 3 5
4 6 2 3 5 1
6 1
0 0 2 3 3
5 4 1 3 2
