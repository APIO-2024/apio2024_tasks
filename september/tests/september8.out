7ckgnn4wyi495puj3ibqf81dqvapyv6b
5
9
5
5
